### ISSUE TITLE
Invaild JSON, if attribute does not exists

### ISSUE THREAD

**Issue by boldt** [telefonicaid/fiware-orion#2596]
I installed an Orion on localhost:1026. I realized, that there is an issue with "Query by attribute content"

## Step 1: Open this URL

http://localhost:1026/v2/entities

This returns a bunch of valid entities.

```
[
  {
    "id": "100",
    "type": "Person",
    "age": {
        "value": "30",
        "type": "Number"
     },
    "name": {
        "value": "Dennis",
        "type": "Text"
     }
  },{
    "id": "101",
    "type": "Person",
    "age": {
        "value": "30",
        "type": "Number"
     },
    "name": {
        "value": "Florian",
        "type": "Text"
     }
  }
]
```
## Step 2: API description

The [API description "Query by attribute content"](https://telefonicaid.github.io/fiware-orion/api/v2/stable/cookbook/) sais:

Despription of parameter `attrs`: Attributes to be returned Example: `name`.

## Step 3: Let's add an "existsing" attribute

Based on Step 1, I'm using an existsing attribute, e.g.,`name`:

http://localhost:1026/v2/entities?attrs=name

This returns a valid JSON with entities containing the attributes `id`, `type` and `name`.

```
[
  {
    "id": "100",
    "type": "Person",
    "name": {
        "value": "Dennis",
        "type": "Text"
     }
  },{
    "id": "101",
    "type": "Person",
    "name": {
        "value": "Florian",
        "type": "Text"
     }
  }
]
```

## Step 4: Let's add a "non existsing" attribute (the issue)

http://localhost:1026/v2/entities?attrs=blabla

This returns  an **invalid** JSON with attributes `id` and `type`, followed by a comma:

```
[
  {
    "id": "100",
    "type": "Person", <<< HERE IS THE ISSUE
  },{
    "id": "101",
    "type": "Person", <<< HERE IS THE ISSUE
  }
]
```

These commas are not allowed within JSON!

## Step 5: Let's add an "existsing" and a "non existsing" attribute

http://localhost:1026/v2/entities?attrs=name,blabla

This returns a valid JSON with entities containing the attributes `id`, `type` and `name`

```
[
  {
    "id": "100",
    "type": "Person",
    "name": {
        "value": "Dennis",
        "type": "Text"
     }
  },{
    "id": "101",
    "type": "Person",
    "name": {
        "value": "Florian",
        "type": "Text"
     }
  }
]
```

**Comment by fgalan** [comment]
Thanks for the feedback!

Which Orion version are you using? I mean, the result of `contextBroker --version` command.

**Comment by boldt** [comment]
We're running multiple versions: 1.2.1 and 1.1.0

**Comment by fgalan** [comment]
It seems that the bug has been fixed since 1.2.0 (probably in Orion 1.3.0, which introduced a lot of fixed).

Could you test with the newest Orion version at this time (1.4.1), please?

**Comment by boldt** [comment]
We updated to 1.4.1 and the issue seems to be fixed.

Thanks @fgalan for the quick support!
